src/test/java/core/basesyntax/service/FruitTransactionParserTest.java

```diff
@@ -0,0 +1,56 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+class FruitTransactionParserTest {
```

Comment:
add negative tests, a lot of things can go wrong during any parsing, does your service uphold your contract/meet your expectations?

Reply:
I added, now I think I checked everything what was possible